howso/engine/project.py

```diff
@@ -226,22 +235,22 @@ def delete(self) -> None:
         self._created = False
         self._id = None
 
-    def _update_attributes(self, project: BaseProject) -> None:
+    def _update_attributes(self, project) -> None:
```

Comment:
Should take a dict (also need to update docstring)

```suggestion
    def _update_attributes(self, project: dict) -> None:
```

Reply:
Should be addressed by: d66654ea10038f70a275bb4aa504128552745e60